UnityProject/Assets/Scripts/UI/Systems/Lobby/CharacterCustomization.cs

```diff
@@ -1094,11 +1134,16 @@ public void SubSaveBodyPart(BodyPart bodyPart, string path)
 			}
 
 
-			if (bodyPart?.OrganStorage?.Populater?.DeprecatedContents != null)
+			if (bodyPart != null && bodyPart.OrganStorage?.Populater?.DeprecatedContents != null)
```

Comment:
the first ``!= null`` isn't needed. The Elvis operator did already what you are doing here.

Reply:
Rider complains about the ? operator on the body part because Unity doesn't like that on gameobjects

Reply:
For objects that extend UnityObject (Game object, components, etc), you use OrNull()  before the Elvis.